async-race/src/store/garageStore.ts

```diff
@@ -0,0 +1,139 @@
+import { defineStore } from "pinia";
+import type { Car, Drive, DriveOptions, Engine } from "../common/model";
+import { getCarBrand, getRandomColor } from "../common/functions";
+import { request } from "../utils/requests.ts";
+import { ref, watchEffect } from "vue";
+
+export const useGarageStore = defineStore("garage", () => {
+  const garage = ref<Car[]>([]);
+  const limit = ref(7);
+  const page = ref(1);
+  const selectedCarId = ref<number | null>(null);
+  const driveOptions = ref<DriveOptions[]>([]);
+  const carDriveStatus = ref<undefined | true>(undefined);
+
```

Comment:
ne nuzhno ukazivat undefined

```suggestion
const carDriveStatus = ref<true>()
```